src/arrow_visualizer.jl

```diff
@@ -0,0 +1,50 @@
+struct ArrowVisualizer{V<:AbstractVisualizer}
+    shaft_vis::V
+    head_vis::V
+end
+
+ArrowVisualizer(vis::AbstractVisualizer) = ArrowVisualizer(vis[:shaft], vis[:head])
+
+function setobject!(vis::ArrowVisualizer, material::AbstractMaterial=defaultmaterial();
+        shaft_material::AbstractMaterial=material,
+        head_material::AbstractMaterial=material)
+    settransform!(vis, zero(Point{3, Float64}), zero(Vec{3, Float64}))
+    shaft = Cylinder(zero(Point{3, Float64}), Point(0.0, 0.0, 1.0), 1.0)
+    setobject!(vis.shaft_vis, shaft, shaft_material)
+    head = Cone(zero(Point{3, Float64}), Point(0.0, 0.0, 1.0), 1.0)
+    setobject!(vis.head_vis, head, head_material)
+    vis
+end
+
+function Base.show(io::IO, v::ArrowVisualizer)
+    print(io, "MeshCat ArrowVisualizer with paths $(v.shaft_vis.path) and $(v.head_vis.path).")
+end
+
+function settransform!(vis::ArrowVisualizer, base::Point{3}, vec::Vec{3};
+        shaft_radius=0.01,
+        max_head_radius=2*shaft_radius,
+        max_head_length=max_head_radius)
```

Comment:
The keyword arguments could also be properties of `ArrowVisualizer`. Do you think that's better?